vendor/ruby/astexport.rb

```diff
@@ -353,7 +379,7 @@ def visit_mrhs_new_from_args(node)
       exps = visit_exps(exps)
       final_exp = visit(final_exp)
     else
-      exp = visit_exps(to_ary(exps))
+      exps = visit_exps(to_ary(exps))
```

Comment:
I think this was just a typo, added failing tests for this code path then they passed after this tweak.
Let me know if I missed something :+1: